Tests/CXOneChatSDKTests/Examples/AgentTypingEnded.json

```diff
@@ -0,0 +1,95 @@
+{
+    "eventId": "eff2fd7d-37b0-42c3-a6f2-e4d2611b5f3b",
+    "eventObject": "SenderAction",
+    "eventType": "SenderTypingEnded",
+    "createdAt": "2024-09-18T21:40:07+01:00",
+    "createdAtWithMilliseconds": "2024-09-18T21:40:07.277+01:00",
+    "data": {
+        "brand": {
+            "id": 6477,
+            "friendlyName": "David Berry",
+            "timezone": "CST6CDT",
+            "brandHash": "4ec545cf34a421ab199e",
+            "tenantId": null,
+            "businessUnitId": null
+        },
+        "channel": {
+            "id": "chat_fd554d68-4c1b-4552-b890-671ab835dcd0",
+            "idOnExternalPlatform": "fd554d68-4c1b-4552-b890-671ab835dcd0",
+            "name": "DBerry Messaging",
+            "channelIntegrationId": "2507eb36-065b-4e41-aca9-c5a07af4803d",
+            "integrationBoxIdentifier": "chat",
+            "realExternalPlatformId": "chat",
+            "externalPlatformId": "chat",
+            "externalPlatformAvatar": "https://channels-assets-qa.brandembassy.com/all/default/external-platform-avatar/chat.png",
```

Comment:
Note:
usage internal domain `channels-assets-qa.brandembassy.com`

Reply:
tied to an internal test case, should be fine